definitions/torrentz2.yml

```diff
@@ -0,0 +1,53 @@
+---
+  site: torrentz2
+  name: Torrentz2
+  language: en-us
+  links:
+    - https://torrentz2.eu/
+
+  caps:
+    categories:
+      "Video Tv": TV
+      "Video": Movies
+      "Video Movie HD": Movies/HD
+
+    modes:
+      search: [f]
+      tv-search: [f, season, ep]
```

Comment:
Ok, I wasn't sure about it but I didn't found doc about creating definitions, I have changed to q but I don't see any difference